apps/indexer/src/handlers/GitcoinAttestationNetwork.ts

```
@@ -0,0 +1,6 @@
+// /*
+//  * Please refer to https://docs.envio.dev for a thorough guide on all Envio indexer features
+//  */
+import { GitcoinAttestationNetwork } from "../../generated";
+
+GitcoinAttestationNetwork.OnAttested.handler(async ({}) => {});
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)